addons/project/tests/test_project_base.py

```diff
@@ -386,3 +386,13 @@ def test_create_task_in_batch_with_email_cc(self):
         self.assertTrue(user_b.partner_id in task_2.message_partner_ids)
         self.assertTrue(user_c.partner_id in task_2.message_partner_ids)
         self.assertFalse(partner in task_2.message_partner_ids)
+
+    def test_create_private_task_in_batch(self):
+        """ This test ensures that copying private task in batch can be done correctly."""
+
+        values = [{'name': 'task 1', 'user_ids': self.env.user.ids, 'project_id': False},
+                  {'name': 'task 2', 'user_ids': self.env.user.ids, 'project_id': False}]
+        tasks = self.env['project.task'].create(values)
+        copied_tasks = tasks.copy()
```

Comment:
```suggestion
        task_0, task_1 = self.env['project.task'].create([{
            'name': f'task {i}',
            'user_ids': self.env.user.ids,
            'project_id': False
        } for i in range(2)]).copy()
```
? :slightly_smiling_face: